test/unittest/dds/xtypes/type_representation/TypeObjectUtilsTests.cpp

```diff
@@ -76,6 +76,40 @@ void register_empty_structure_type_object()
     TypeObjectUtils::build_and_register_struct_type_object(struct_type, empty_struct_name);
 }
 
+void register_plain_seq_type_object(
+        eprosima::fastcdr::external<TypeIdentifier> complete_typeid)
+{
+    std::string plain_seq_name = "plain_sequence";
+    CollectionElementFlag flags = TypeObjectUtils::build_collection_element_flag(
+        eprosima::fastdds::dds::xtypes::TryConstructKind::USE_DEFAULT, false);
+    PlainCollectionHeader header = TypeObjectUtils::build_plain_collection_header(EK_COMPLETE, flags);
+
+
+    eprosima::fastcdr::external<TypeIdentifier> primitive_identifier{new TypeIdentifier()};
+    primitive_identifier->_d(TK_INT32);
```

Comment:
Why are you declaring this TypeIdentifier if finally it is not being used?

Reply:
removed in [d3245e1](https://github.com/eProsima/Fast-DDS/pull/4496/commits/d3245e16dd687c62ce397be8018c718828d2bc6e)